stellargraph/data/explorer.py

```diff
@@ -780,3 +783,166 @@ def _check_neighbourhood_sizes(self, in_size, out_size):
             self._raise_error(
                 "The number of hops for the in and out neighbourhoods must be the same."
             )
+
+
+@experimental(reason="requires more thorough testing and documentation")
+class TemporalRandomWalk(GraphWalk):
+    """
+    Performs uniform temporal random walks on the given graph
```

Comment:
Should this explain that a "given graph" should have edge weights that correspond to timestamps and the format and interpretation of said timestamps? Are these expected to be seconds, minutes, hours, or another format, e.g., HH:MM:SS. 

If such an explanation is available elsewhere, then ignore this comment.

Reply:
Good point, currently we just expect edge weights to be numbers that represent time rather than some kind of timestamp string. I think the exact units (minutes or hours, etc) don't actually matter here, as long as all the time values can be compared relative to each other. I'll add a comment